### PR TITLE
[Task] Set up custom string logic for calling screen

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerView.kt [hunk 1]
```diff
@@ -68,13 +68,13 @@ internal class BannerView : ConstraintLayout {
                 BannerInfoType.RECORDING_STOPPED_STILL_TRANSCRIBING,
                 BannerInfoType.TRANSCRIPTION_STOPPED_STILL_RECORDING,
                 -> {
-                    context.getString(R.string.azure_communication_ui_calling_view_link_privacy_policy_url)
+                    getLocalizedString(R.string.azure_communication_ui_calling_view_link_privacy_policy_url)
```
Comment (Mohtasim) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerView.kt:71]: Shouldn't we pass the context also with the hash int of resource string id?
As the getLocalizedString signature is 'getLocalizedString(context: Context, stringKey: Int)'

Comment (ShaunaSong) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerView.kt:71]: getLocalizedString() have the other reuse method end of this class

Comment (Mohtasim) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerView.kt:71]: I see. at line 129. 
This way is much cleaner. Good one. Thanks

**Review by Mohtasim — APPROVED** [review]
Looks good to me

**Review by lpeixin — APPROVED** [review]

**Pull request merged**